plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/datatools/ui/wizard/ConnectionProfileDatabasePage.java

```diff
@@ -10,15 +10,26 @@
  ******************************************************************************/ 
 package org.jboss.reddeer.eclipse.datatools.ui.wizard;
 
+import org.eclipse.swt.widgets.Shell;
+import org.jboss.reddeer.common.exception.RedDeerException;
+import org.jboss.reddeer.common.exception.WaitTimeoutExpiredException;
+import org.jboss.reddeer.common.wait.WaitWhile;
+import org.jboss.reddeer.core.condition.WidgetIsFound;
+import org.jboss.reddeer.core.matcher.WithTextMatcher;
 import org.jboss.reddeer.jface.wizard.WizardPage;
+import org.jboss.reddeer.swt.condition.ShellIsAvailable;
+import org.jboss.reddeer.swt.impl.button.CheckBox;
+import org.jboss.reddeer.swt.impl.button.OkButton;
 import org.jboss.reddeer.swt.impl.combo.LabeledCombo;
+import org.jboss.reddeer.swt.impl.shell.DefaultShell;
+import org.jboss.reddeer.swt.impl.text.LabeledText;
 
 /**
  * An abstract wizard page for database connection profile.
  * 
  * IMPORTANT: It is assumed that the appropriate driver definition has been already created.
  * 
- * @author apodhrad
+ * @author apodhrad, mlabuda@redhat.com
  * 
  */
 public abstract class ConnectionProfileDatabasePage extends WizardPage {
```

Comment:
name & package is wrong. Should be eclipse.datatools.connectivity.ui.wizards.ExtensibleProfileDetailsWizardPage